src/Shell/SenderShell.php

```diff
@@ -68,6 +69,9 @@ public function main()
             $layout = $e->layout === 'default' ? $this->params['layout'] : $e->layout;
             $headers = empty($e->headers) ? array() : (array) $e->headers;
             $theme = empty($e->theme) ? '' : (string) $e->theme;
+            
+            if (!empty($e->template_vars['language']))
```

Comment:
I would name it `locale`

Also this is missing the opening and losing curly braces `if (...) { ... }`